CONTRIBUTING.md

```diff
@@ -1,74 +1,52 @@
 # Contribution policy
-A “Contribution” is any work that is voluntarily submitted to a project. This work may be code, documentation, design, answering questions, or even triaging issues.
```

Comment:
I believe the first section was auto-generated by CELA on creation of the repository.  I don't think you changed it materially enough, but, wanted to call this out since i'm not a lawyer.  Also, we do still refer to the link for CLA.

Reply:
I think it is fine.